src/prebid.js

```diff
@@ -260,21 +260,21 @@ $$PREBID_GLOBAL$$.getBidResponsesForAdUnitCode = function (adUnitCode) {
  * Set query string targeting on all GPT ad units.
  * @alias module:$$PREBID_GLOBAL$$.setTargetingForGPTAsync
  */
-$$PREBID_GLOBAL$$.setTargetingForGPTAsync = function (adUnits) {
+$$PREBID_GLOBAL$$.setTargetingForGPTAsync = function (adUnitsCodeArr) {
```

Comment:
Do you think we should start JSDoc-ing params whenever the types cause confusion?

Renaming it here does help... but in other cases (like `getAllTargeting`, where the arg might be a string or array) that's not always possible.